napari/layers/points/points.py

```diff
@@ -756,7 +756,18 @@ def _extent_data(self) -> np.ndarray:
             maxs = np.max(self.data, axis=0)
             mins = np.min(self.data, axis=0)
             extrema = np.vstack([mins, maxs])
-        return extrema
+        return extrema.astype(float)
+
+    @property
+    def _extent_data_augmented(self):
+        extent = self._extent_data
```

Comment:
It's not entirely clear that this is a property and therefore that `extent` will be a copy (?) that it is safe to modify. Therefore, I suggest adding a comment here to clarify that.

```suggestion
        # _extent_data is a property that returns a new/copied array, which
        # is safe to modify below
        extent = self._extent_data
```

Reply:
In general, I think a lot of this confusion would be solved by not using properties at all for these relatively expensive attributes, and instead use a method.

Reply:
Ok but in the meantime commit the comment?